exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -125,11 +139,13 @@ def __init__(
         self.ipv6 = ipv6
         self.retry = retry
         self.max_tag_value_length = max_tag_value_length
+        self.transport_format = transport_format
```

Comment:
Should allow this to be read from env var just like the URL. I suggest using`OTEL_EXPORTER_ZIPKIN_TRANSPORT` env config var.

Reply:
+1, adding.